libs/crypto/keys_manager_impl.hpp

```diff
@@ -28,14 +17,30 @@ namespace iroha {
 
   class KeysManagerImpl : public KeysManager {
    public:
-    explicit KeysManagerImpl(const std::string &account_name);
+    /**
+     * Initialize key manager for a specific account
+     * @param account_id - fully qualified account id, e.g. admin@test
+     * @param path_to_keypair - path to directory that contains priv and pub key
+     * of an account
+     *
+     * Here we use an empty string as a default value of path to file, since
+     * there are usages of KeysManagerImpl with path passed as a part of
+     * account_id.
+     */
+    explicit KeysManagerImpl(
+        const std::string &account_id,
+        const boost::filesystem::path &path_to_keypair = "");
 
-    bool createKeys() override;
-    bool createKeys(const std::string &pass_phrase) override;
+    bool createKeys(const std::string &pass_phrase = "") override;
 
-    boost::optional<shared_model::crypto::Keypair> loadKeys() override;
+    /**
+     * Loads keypair from disk
```

Comment:
It should not be requred to write the whole doc if the behavior is already defined here:
https://github.com/hyperledger/iroha/blob/63cb6a186dccb3e40f222af3da561998a05a4f5d/libs/crypto/keys_manager.hpp#L47-L54